goFeed.sh

```diff
@@ -9,6 +9,21 @@ else
   exit 1
 fi
 
+getGoVulnCheck() {
```

Comment:
@JalexChen can you review this section as you are the most knowledgeable with regards to these scripts you set up previously, thanks :)